quick-start.md

```diff
@@ -69,27 +70,45 @@ export INITIUM_CLUSTER_CA_CERT=$(kubectl get secrets initium-cli-token -o jsonpa
 ```
 cd initium-nodejs-demo-app
 git checkout -b initium-test
-initium-cli --app-name "demo-app" init config > .initium.yaml
+initium-cli init config > .initium.yaml
 initium-cli init github
 ```
 
-6. Commit the changes and open a PR
+6. Commit the changes and open a PR. Please use the following values in order to be aligned with the commands on the next steps:
```

Comment:
I'm thinking that we should add this in the `demo-app` repo...

Let me try that